specs/rest-api.md

```diff
@@ -665,9 +665,17 @@ in an optimized format.
 Output:
 
     {
+        'aliases': ['alias1',...],
+	'architecture': 0,
+	'fingerprint': a3625aeada09576673620d1d048eed7ac101c4c2409f527a1f77e237fa280e32,
         'properties': {
             'key': 'value'
         },
+	'public': 1,
```

Comment:
that's supposed to be a bool in the rest API, not an int, so that'd be "true" here